src/compiler/transformers/test/proxy-custom-element-function.spec.ts

```diff
@@ -1,6 +1,6 @@
 import * as d from '@stencil/core/declarations';
 import { mockCompilerCtx } from '@stencil/core/testing';
-import * as ts from 'typescript';
```

Comment:
I think I'm skimming over the reason for this change in the diff w/o realizing (need ☕) -  for my own knowledge, why did we need to change this?

Reply:
I think that just snuck in, I'll remove it